src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -649,12 +645,14 @@ private void spawnCitizen(CitizenData data)
         {
             EntityCitizen entity = new EntityCitizen(world);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'entity' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)